operatorcourier/cli.py

```diff
@@ -67,7 +67,8 @@ def parse(self):
         verify_parser.add_argument(
             '--validation-output',
             dest='validation_output',
-            help='A file to write validation warnings and errors to in JSON format')
+            help='A file to write validation warnings and errors to in JSON format.'
+                 'This option only works with the flat directory structure.')
```

Comment:
Why validation input cannot be saved to file?

Reply:
Because the nested directory isn't generating a bundle at all. It actually zips up the directory itself, intact. We aren't generating yaml in this case, so there's no file to be generated.

Reply:
But this is validation output, there is no validation for nested structure?

Reply:
Sorry I think I misunderstood which output file we were referring to here. It seems like this should work on both.